spec/features/reservations_spec.rb

```diff
@@ -744,9 +757,9 @@
   context 'invalid items on confirm page' do
     before(:each) do
       empty_cart
-      # change items per page from 1 to 10 so second item shows up
-      select('10', from: 'items_per_page')
       find('#items_per_form').submit_form!
```

Comment:
I think we can delete this line as well since we're not changing that setting manually anymore.